movie-app/src/Components/MovieCard.js

```diff
@@ -1,14 +1,34 @@
+
+import {useState} from 'react';
+
 const MovieCard = ({movie}) => {
+  const moviePoster = {
+    backgroundImage: `url(${movie.Poster})`,
+  }
+
+  const [cardHover, setCardHover] = useState(false);
+// when true, apply show info animation
+  const showInfoOnHover = () => {
+    setCardHover(!cardHover)
+  }
+
+  const hideInfoOnHover = () => {
+    setCardHover(!cardHover)
+  }
+
   return (
-    <div>
-      <img src={movie.Poster} alt={`${movie.Title} poster`}/>
-      <h1>{movie.Title}</h1>
-      <h3>{movie.Year}</h3>
-    </div>
-  );
-};
+    <div className="card" style={moviePoster} onMouseEnter ={showInfoOnHover} onMouseLeave={hideInfoOnHover}>
 
+      <div className={`info-container ${cardHover? `hide-info`: `show-info`}`}>
+        <h2>{movie.Title}</h2>
+        <h3>{movie.Year}</h3>
```

Comment:
These two can have the same heading level, as they're below an `h1` and Year is not necessarily a subsection of Title.